compiler/rustc_ty_utils/src/layout.rs

```diff
@@ -18,8 +23,9 @@ use rustc_middle::ty::{
 use rustc_session::{DataTypeKind, FieldInfo, FieldKind, SizeKind, VariantInfo};
 use rustc_span::sym;
 use rustc_span::symbol::Symbol;
-use rustc_target::abi::*;
+use rustc_target::abi::{FIRST_VARIANT, FieldIdx, Layout, VariantIdx};
```

Comment:
...and these are the reason I bothered.

Reply:
Hmm, factoring these out will also be a bit of a pain. Ah well, will worry about it later.